gtest_parallel.py

```diff
@@ -974,7 +974,11 @@ def _process_args(self, largs, rargs, values):
             try:
                 optparse.OptionParser._process_args(self, largs, rargs, values)
             except (optparse.BadOptionError,optparse.AmbiguousOptionError) as err:
-                self.eargs.append([s for s in copy_args if err.opt_str in s])
+                eargs =  [s for s in copy_args if err.opt_str in s][0]
+                self.eargs.append(eargs)
+                # Skip the next arg if the current arg has an `=`
+                if rargs and "=" in eargs:
```

Comment:
Just making sure I'm understanding this correctly: why are we checking for `=` skipping the next arg here? Or, maybe better question: what problem does this fix?

Reply:
The rargs starts with something like `--data=~/runtime/data`. The args are split if they contain an `=` and is turned into 2 options in the array, `--data` and `~/runtime/data`. If an exception is thrown on the first option (`--data`) the 2nd option (`~/runtime/data`) is added to the [binaries](https://github.com/Esri/gtest-parallel/blob/runtimecore/gtest_parallel.py#L1037) list. The script tries to run the extra binary and can't find it. Besides removing the next option, I added a [check](https://github.com/Esri/gtest-parallel/blob/craig/additional_cmds/gtest_parallel.py#L1045) to make sure parsing happens correctly or else a good error message will be returned.

Reply:
Ah, good deal. I didn't realize that `=` would split the arguments.